File: Source/root/src/main/java/root/communication/messages/ReplayServerResponse.java
package root.communication.messages;

import java.util.List;

public class ReplayServerResponse {

	public ReplayResponseStatus status;

	public List<GameDetails> games;

	public ReplayServerResponse(ReplayResponseStatus status) {
		this.status = status;
		this.games = null;
	}

	public ReplayServerResponse(ReplayResponseStatus status, List<GameDetails> games) {
		this.status = status;
		this.games = games;
	}

}

File: Source/server/rabbit-game/src/SharedModel/ReplayServerResponse.cs

namespace RabbitGameServer.SharedModel
{
	public enum ReplayResponseStatus
	{
		SUCCESS,
		INVALID_NAME,
		SERVER_ERROR
	}

	public class ReplayServerResponse
	{
		public ReplayResponseStatus status{get;set;}
		public List<GameDetails> games{get;set;}

		public ReplayServerResponse(ReplayResponseStatus status, List<GameDetails> games)
		{
			this.status = status;
			this.games = games;
		}
	}
}